hail/src/main/scala/is/hail/linalg/BlockMatrix.scala

```diff
@@ -117,7 +117,7 @@ object BlockMatrix {
       val iOffset = i * blockSize
       val jOffset = j * blockSize
 
-      HailContext.backend.broadcast(lm(iOffset until iOffset + blockNRows, jOffset until jOffset + blockNCols).copy)
+      HailContext.backend.broadcast(lm(iOffset until iOffset + blockNRows, jOffset until jOffset + blockNCols))
```

Comment:
Copy function not working as expected --> applies operation for first block in block matrix and returns the result of that for all blocks.

Could be related to this:
https://github.com/scalanlp/breeze/issues/772